AI_Agents_Guide/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Guide to Deploying AI Agents with Triton Inference Server
+
+Welcome to the **Guide to Deploying AI Agents with Triton Inference Server**.
+This repository provides a set of tutorials designed to help you deploy
+AI agents efficiently using the Triton Inference Server. This guide is intended
+for users who are already familiar with the basics of Triton and are looking to
+expand their knowledge.
+
+For beginners, we recommend starting with the
+[Conceptual Guide](tutorials/Conceptual_Guide/README.md), which covers
+foundational concepts and basic setup of Triton Inference Server.
+
+## AI agents and Agentic Workflows
+
+Modern large language models (LLMs) are integral components of AI agents —
```

Comment:
are there examples of AI agents which integrate LLMs? wondering if some high level understanding of AI agents would be good to add.

Reply:
you mean real-world examples, or description, or both?

Reply:
both. but can happen as an additional work later on. approving for merge for first version of documentation.

Reply:
Created ticket: DLIS-7291